data/io.elementary.appcenter.appdata.xml.in

```diff
@@ -101,8 +101,7 @@
         <release version="0.1.0" date="2016-08-15"/>
     </releases>
     <provides>
-        <binary>appcenter</binary>
-        <binary>appcenter-daemon</binary>
```

Comment:
Yeah, as far as I can tell `appcenter-daemon` isn't a binary that exists anymore.